astropy/units/tests/test_logarithmic.py

```diff
@@ -77,18 +77,20 @@ def test_subclass_creation(self, lu_cls, physical_unit):
 
 
 def test_predefined_magnitudes():
+    # these magnitudes are only approximations to the true definition
     assert_quantity_allclose((-21.1*u.STmag).physical,
-                             1.*u.erg/u.cm**2/u.s/u.AA)
+                             1.*u.erg/u.cm**2/u.s/u.AA, rtol=.1)
     assert_quantity_allclose((-48.6*u.ABmag).physical,
-                             1.*u.erg/u.cm**2/u.s/u.Hz)
+                             1.*u.erg/u.cm**2/u.s/u.Hz, rtol=.1)
```

Comment:
With the `AB` system, I'm fairly sure I just went back to the original Oke & Gunn paper....